freemarker-wrapper/pom.xml

```diff
@@ -5,8 +5,8 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>freemarker-wrapper</artifactId>
-    <groupId>io.codeontap</groupId>
-    <version>1.11.1</version>
+    <groupId>io.hamlet</groupId>
+    <version>1.12.1-rc10</version>
```

Comment:
Does this need updating prior to formal release?

Reply:
@ml019, I update the version in the pom.xml file.